export.py

```diff
@@ -0,0 +1,251 @@
+        existence_counter = 0
+        while self.name.exists():
+            existence_counter += 1
+            self.name = self.name.parent / f"{filename}-{existence_counter}.onnx"
+        return self.name
+
+
+def parse_args() -> ExportArgs:
+    """
+    Add and parse arguments for exporting a ViT model to ONNX
+    :return: A ExportArgs Object with typed arguments
+    """
+    parser = argparse.ArgumentParser(description="Export ViT models to ONNX")
+
+    parser.add_argument(
+        "--checkpoint",
+        type=str,
+        required=True,
+        help="The ViT pytorch checkpoint to export",
+    )
+
+    parser.add_argument(
+        "--config",
+        "-c",
+        type=str,
+        help="The config used to train the ViT model. By default "
+        "the parser will attempt to open args.yaml in the same "
+        "directory as the checkpoint",
+    )
+
+    parser.add_argument(
+        "--recipe",
+        "-r",
+        type=str,
+        default=None,
+        help="Path or SparseZoo stub to the recipe used for training, "
+        "omit if no recipe used. If no recipe given, "
+        "the checkpoint recipe will be applied if present.",
+    )
+
+    parser.add_argument(
+        "--no-qat",
+        "-N",
+        action="store_true",
+        help="Flag to prevent conversion of a QAT(Quantization Aware Training) "
+             "Graph to a Quantized Graph",
+    )
+
+    parser.add_argument(
+        "--batch-size",
+        "-b",
+        type=int,
+        default=1,
+        help="The batch size to use while exporting the Model graph to ONNX;"
+        "Defaults to 1",
+    )
+
+    parser.add_argument(
+        "--image-shape",
+        "-S",
+        type=int,
+        nargs="+",
+        default=(3, 550, 550),
+        help="The image shape in (C, S, S) format to use for exporting the "
+        "Model graph to ONNX; Defaults to (3, 550, 550)",
+    )
+
+    parser.add_argument(
+        "--name",
+        "-n",
+        type=str,
+        default=None,
+        help="The path to use for saving the exported ONNX model."
+             "If directory, defaults to checkpoint-name.onnx",
+    )
+
+    args = parser.parse_args()
+    return ExportArgs(**vars(args))
+
+
+def export(args: ExportArgs):
+
+    # Load model-specific configs
+    with open(args.config, 'r') as f:
+            cfg = yaml.safe_load(f)
+            cfg = Namespace(**cfg)
+                
+    model = create_model(
+            cfg.model,
+            pretrained=cfg.pretrained,
+            num_classes=cfg.num_classes,
+            drop_rate=cfg.drop,
+            drop_connect_rate=cfg.drop_connect, 
+            drop_path_rate=cfg.drop_path,
+            drop_block_rate=cfg.drop_block,
+            global_pool=cfg.gp,
+            bn_tf=cfg.bn_tf,
+            bn_momentum=cfg.bn_momentum,
+            bn_eps=cfg.bn_eps,
+            scriptable=cfg.torchscript,
+            checkpoint_path=None)
```

Comment:
nit: closing brace on newline? Comma after checkpoint path? Something like the following:
```suggestion
            checkpoint_path=None,
            )
```